explore/src/main/scala/explore/HelpBody.scala

```diff
@@ -71,9 +69,6 @@ object HelpBody:
       .useEffectOnMountBy { (props, _, state) =>
         load(props.url).flatMap(v => state.set(Pot.fromTry(v)).to[IO])
       }
-      .useGlobalHotkeysBy((_, helpCtx, _) =>
-        UseHotkeysProps(CloseHelp.value, helpCtx.displayedHelp.set(none))
```

Comment:
Is the `CloseHelp` hotkey being used anywhere else? If not, can it be deleted?

Reply:
You're right. I'll delete it. prime handles `closeOnEsc` by itself

Reply:
You can do `state.setAsync` (requires `import crystal.react.implicits.*`).